apps/asap-server/test/handlers/calendar/subscribe-handler.test.ts

```diff
@@ -301,3 +263,12 @@ describe('Unsubscribing', () => {
     expect(nock.isDone()).toBe(true);
   });
 });
+
+const getEvent = (
```

Comment:
I'd rather see this before the tests

Reply:
you would? I kinda prefer to have all the fixtures and generators either in a different file or at the bottom. I can move to another file actually